queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -206,12 +215,16 @@ def specify_instrument_directories(self,
         # Specify directories where autoreduction output will go
         return temporary_directory + instrument_output_directory
 
-    def reduction_started(self):
-        """Log and update AMQ message to reduction started"""
-        logger.debug("Calling: %s\n%s",
-                     ACTIVEMQ_SETTINGS.reduction_started,
-                     self.message.serialize(limit_reduction_script=True))
-        self.client.send(ACTIVEMQ_SETTINGS.reduction_started, self.message)
+    def create_log_path(self, file_name_with_extension, log_directory):
+        """Create log file and place in reduction_log_directory
+      :param file_name_with_extension: (string) file name and extension type
+      :param log_directory: (str) log directory path
+      :return: (str) log file path
+      """
+
+        log_and_err_name = f"RB{self.proposal}Run{self.run_number}"
```

Comment:
Could we put underscores between the RB/Run and the numbers so we can search through?

Reply:
I agree, this would definitely make it more readable.
@ElliotAOram would you be happy with this change?

Reply:
Agreed